lib/fizzy/float_handling.hpp

```diff
@@ -0,0 +1,18 @@
+
+#pragma once
+namespace fizzy
+{
+union Num32
+{
+    unsigned int i32;
+    float f32;
+};
+
+inline float abs(float input) noexcept
```

Comment:
No need to place this function is separate file. Move it to `execute.cpp`, next to functions `fceil`, `ffloor`, etc, in the order of the opcodes.